src/v/kafka/server/handlers/handler_probe.cc

```diff
@@ -32,7 +32,7 @@ handler_probe_manager::handler_probe_manager()
   , _probes(max_api_key() + 2) {
     const auto unknown_handler_key = max_api_key() + 1;
     for (size_t i = 0; i < _probes.size(); i++) {
-        auto key = api_key{i};
+        auto key = api_key(i);
```

Comment:
question: why prefer this change over a static cast?

Reply:
> question: why prefer this change over a static cast?

Because I was being lazy, but it's a fair point.